types/react-select/src/types.d.ts

```diff
@@ -59,12 +49,17 @@ export interface CommonProps<
      */
     getStyles: (name: string, props: any) => {};
     getValue: () => OptionsType<OptionType>;
+    /** Whether the value container currently holds a value. */
     hasValue: boolean;
+    /** Set when the value container should hold multiple values */
     isMulti: boolean;
+    /** Whether the text is right to left */
+    isRtl: boolean;
```

Comment:
I take it `CommonProps` is used with `Optional<>` to make all of these possible undefined?

Reply:
`CommonProps` are not props on the Select component, they are props that are passed down to all of the sub-components and they will always be defined.